From 0bdc4a0ddc8b809af92adfe7c59d218346d17249 Mon Sep 17 00:00:00 2001
From: Gwenael Treuveur <gwenael.treuveur@foss.st.com>
Date: Tue, 10 Sep 2024 14:01:57 +0200
Subject: [PATCH 1135/1695] dts: stm32: change default d3 domain source clock
 on stm32mp257f-dk

Change the default source clock to MSI kernel clock.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 7890c55d2e9bc6c96d6e3a7924dc5b3bf7040ced dts: stm32: change default d3 domain source clock on stm32mp257f-dk]
Change-Id: I99067de45e1ae380bfdc397871d96152b9d5fa06
Signed-off-by: Gwenael Treuveur <gwenael.treuveur@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/403336
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
Tested-by: Gwenael TREUVEUR <gwenael.treuveur@st.com>
Reviewed-by: Gwenael TREUVEUR <gwenael.treuveur@st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
---
 core/arch/arm/dts/stm32mp257f-dk-ca35tdcid-rcc.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/arch/arm/dts/stm32mp257f-dk-ca35tdcid-rcc.dtsi b/core/arch/arm/dts/stm32mp257f-dk-ca35tdcid-rcc.dtsi
index bd4edcc7c..e935534b4 100644
--- a/core/arch/arm/dts/stm32mp257f-dk-ca35tdcid-rcc.dtsi
+++ b/core/arch/arm/dts/stm32mp257f-dk-ca35tdcid-rcc.dtsi
@@ -109,7 +109,7 @@
 		MUX_CFG(MUX_LVDSPHY, MUX_LVDSPHY_FLEX32)
 		MUX_CFG(MUX_DTS, MUX_DTS_HSE)
 		MUX_CFG(MUX_RTC, MUX_RTC_LSE)
-		MUX_CFG(MUX_D3PER, MUX_D3PER_LSE)
+		MUX_CFG(MUX_D3PER, MUX_D3PER_MSI)
 		MCO_CFG(MCO1, MUX_MCO1_FLEX61, MCO_OFF)
 		MCO_CFG(MCO2, MUX_MCO2_FLEX62, MCO_OFF)
 	>;
-- 
2.47.2

